### PR TITLE
{data}[foss/2020a] harmony v1.0.0-20200224

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3416.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/a3e9553638693a0559eaef2f5291be90 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2606.swalot.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/8576075d21a764def4940fc855bc7feb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/3f4f4e71f04fb1579e4319b12e2e566e for a full test report.

File: easybuild/easyconfigs/h/harmony/harmony-1.0.0-20200224-foss-2020a-R-4.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+name = 'harmony'
+local_commit = '88b1e2a'
+# see DESCRIPTION to determine version,
+# but also take date of last commit into account (since version isn't always bumped)
+version = '1.0.0-20200224'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://portals.broadinstitute.org/harmony'
+description = "Harmony is a general-purpose R package with an efficient algorithm for integrating multiple data sets."
```
Comment (smoors) [inline easybuild/easyconfigs/h/harmony/harmony-1.0.0-20200224-foss-2020a-R-4.0.0.eb:11]: can you be any more vague than that?? :)

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node318.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/c8be95c891360e06fb392b38c3baa6d0 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**